sdk/workflow-tengo/src/exec/index.lib.tengo

```diff
@@ -698,7 +738,11 @@ builder := func() {
 				assets: smart.createMapResource(assetInfos),
 				assetDescriptors: assetDescriptors,
 				assetRules: assetRules,
-				assetCache: assetCache
+				assetCache: assetCache,
+				quota: smart.createJsonResource({
```

Comment:
We might want to add quota only when it was explicitly set, so the old exec templates won't be recalculated because their inputs have changed. What do you think?

Reply:
I put it into separate service field to not affect deduplication at all.